### ISSUE TITLE
videoJS causing tons of automated clicks

### ISSUE THREAD

**Issue by mloveless1227** [videojs/video.js#4771]
Ever since installing JWPlayer for my Google IMA video ads (per documentation), I have noticed an insane increase in Clicks and they seem to be automated somehow. For instance, on around 2500 video ads served yesterday, I had 933 clicks. That's just not right. I reached out to my provider and they can't notice anything unusual. I had been using JWPlayer for the previous 2 years with the same video tag and distribution setup, never seen anything like this. But over the past week I'm averaging like a 40% click through rate and that can't be right.

Can you think of anything that might be causing this? My setup is very basic. Just my google ima tag, a short video, no custom controls and stuff, autoplay.

**Comment by misteroneill** [comment]
I'm confused, is this an issue with Video.js or JWPlayer? If the latter, which it sounds like, you would be better served by reporting it to [JWPlayer](https://github.com/jwplayer).

**Comment by mloveless1227** [comment]
It may or may not be a VideoJS issue, but it started happening at the same time I changed over to VideoJS, as where it was not happening prior with JWPlayer. I did a bit of searching around but I thought it might be beneficial to ask here if this was something that is known to happen if some settings were done incorrectly. That was my main point of posting here. Sorry if this is the wrong place. If it's not relevant and can't be caused by some setting in VideoJS, feel free to close the comment. I just wanted to do everything I could to find out what was happening.
